FrameStreamSockInput.go

```diff
@@ -49,7 +55,7 @@ func (input *FrameStreamSockInput) ReadInto(output chan []byte) {
 			log.Printf("net.Listener.Accept() failed: %s\n", err)
 			continue
```

Comment:
This should be fixed, as a separate issue.

(It will take digging into the possible errors returned by Accept() and knowing which ones we can safely continue on).